### ISSUE TITLE
When sequence data is not provided, error gracefully

### ISSUE THREAD

**Issue by tavinathanson** [hammerlab/pyensembl#106]
Right now, when sequence data is not provided, we just run into an undefined variable.

For example, in `transcript.py` (stolen from @iskandr):

```
if self.sequence is None:
    raise ValueError("No 3' UTR sequence for %s" % self)
```

**Comment by tavinathanson** [comment]
We should also error gracefully when GTF data is not provided.

**Comment by tavinathanson** [comment]
Fixed by #108
